README.md

```diff
@@ -50,35 +52,37 @@ manually.
 Supported Ruby versions
 -----------------------
 
-The factory_girl 3.x+ series supports MRI Ruby 1.9. Additionally, factory_girl
+The factory_bot 3.x+ series supports MRI Ruby 1.9. Additionally, factory_bot
 3.6+ supports JRuby 1.6.7.2+ while running in 1.9 mode. See [GETTING_STARTED]
 for more information on configuring the JRuby environment.
 
-For versions of Ruby prior to 1.9, please use factory_girl 2.x.
+For versions of Ruby prior to 1.9, please use factory_bot 2.x.
```

Comment:
All changes to this “Supported Ruby versions” section are a bit confusing, since there is currently only a 4.x release of factory_bot.